subsys/bluetooth/controller/Kconfig

```diff
@@ -220,7 +220,7 @@ config BT_CTLR_ISO_RX_BUFFERS
 config BT_CTLR_ISO_TX_BUFFERS
 	int "Number of Isochronous Tx buffers"
 	depends on BT_CTLR_ADV_ISO || BT_CTLR_CONN_ISO
-	range 1 19
+	range 1 32
```

Comment:
We need 31 at present so that we can send all fragments in a burst, the previous maximum was 19 so we increased it to accommodate what we require. If the maximum is higher, I don't know why it was set at 19.

Reply:
Should we not just set the maximum to 0xFF now then?

Reply:
Done.